src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```
@@ -425,12 +467,63 @@ def options_dialog(show_gui=True) -> dict[str, Any]:
             [BIP_AND_REFERENCES, ONLY_BIP],
         ),
     ]
-    selections = lux.getInputDialog(
+
+    file_selections = lux.getInputDialog(
         title="AWS Deadline Cloud Submission Options",
-        values=dialog_items,
+        values=file_dialog_items,
         id=DEADLINE_CLOUD_DIALOG_ID,
     )
 
+    cameras = lux.getCameras()
+    active_camera = lux.getCamera()
+
+    if len(cameras) == 1:
+        file_selections["selected_cameras"] = [cameras[0]]
+        return file_selections
+
+    render_dialog_items: list[tuple[Any, ...]] = []
+
+    if cameras:
+        render_dialog_items.append((lux.DIALOG_LABEL, "Select cameras to render:"))
+
+        for i in range(len(cameras)):
+            camera = cameras[i]
+            label = "Current Active Camera" if camera == "last_active" else f"Camera: {camera}"
+
+            is_selected = (camera == active_camera) or (
+                active_camera == "active" and camera == "last_active"
+            )
+
+            render_dialog_items.append(
+                (
+                    f"camera_{i}",
+                    lux.DIALOG_CHECK,
+                    label,
+                    is_selected,
+                )
+            )
+
+    if not render_dialog_items:
+        return file_selections
+
+    render_selections = lux.getInputDialog(
+        title="Select Cameras to Render",
+        values=render_dialog_items,
+    )
+
+    selections = file_selections
+
+    selected_cameras = []
+    for i in range(len(cameras)):
+        key = f"camera_{i}"
+        if key in render_selections and render_selections[key]:
+            selected_cameras.append(cameras[i])
+
+    if not selected_cameras and cameras:
+        selected_cameras = ["last_active"]
```

Should we raise an error if the user does not select any cameras instead?